data/sheragi/archaeology missions.txt

```diff
@@ -300,6 +300,7 @@ mission "Deep Archaeology 5"
 
 ship "Timer Ship"
 	attributes
+		category "Game Mechanics"
```

Comment:
```suggestion
		category "Unclassified"
```

Reply:
Same as with the Ion Storm, this is explicitly used for a mechanic, not a ship classification.